### ISSUE TITLE
Add GUI setup option for LUKS encryption options

### ISSUE THREAD

**Issue by Kenny606** [QubesOS/qubes-issues#6509]
**The problem you're addressing (if any)**
Lack of option in GUI setup to specify Encryption Method requiring manual partition setup from console.

**Describe the solution you'd like**
Ideally a simple drop down during setup offering an option for say aes-256 sha-512, allowing you to override the default encryption inherited from CryptSetup.
For someone who doesn't understand the implication it a simple warning that increasing encryption beyond the default will affect performance should be warning enough I would think for those that don't know that.
I understand there are lots of combinations but if someone has something very specific in mind I would think they should have no problem with the console method.

**Where is the value to a user, and who might that user be?**
In my case I'm versed in virtualization however I am somewhat a linux novice, it just hasn't been a large part of my career.
I would think others interested in trying this OS for privacy and security may in a similar position and might want heavier encryption without going through the console just to really modify two parameters in my case.

**Describe alternatives you've considered**
Without going through the trouble of a GUI perhaps thinking it might negatively impact users overall experience the process could still be implemented, made much simpler than it is now and should be relatively simple to implement using two variables.
For example if I went to the console and set one or both of these variables setup could check for valid values during the creation of the LUKS volume.
hash=sha512
keylength=512
(By default these variables would be undeclared.)

If the values are declared and valid during the partitioning process they would trigger additional parameters to be issued while creating the LUKS volume. If they were declared but contain invalid values simple warn the user the default encryption method will be used. 
It's easily documented, should be simple enough for anyone to set without any trouble when they start setup.
And someone without much concept on the performance impacts won't select a GUI option not knowing the true impact of or understanding the security implication.

Thank you for your consideration.

**Comment by andrewdavidwong** [comment]
Reasons for the "minor" and "help wanted" labels:

- The defaults are already secure enough (if anyone disputes this, they should file a report with LUKS and/or Fedora to implement better defaults, since we merely inherit their defaults).
- Since the defaults are already fine, anyone who wants to modify them should be willing and able to follow the current steps.
- Development time would be better spent on things that significantly improve user security.

**Comment by ben-grande** [comment]
I agree, this is something for upstream Anaconda or LUKS. We don't developer the installer nor the encryption tool. Closing until someone works on this.
